app/presenters/schools/courses/curriculum_presenter.rb

```diff
@@ -66,16 +67,24 @@ def targets
             id: target.id,
             target_group_id: target.target_group_id,
             title: target.title,
-            sortIndex: target.sort_index,
+            sort_index: target.sort_index,
             visibility: target.visibility
           }
         end
       end
 
       def vimeo_access_token?
-        token = @course.school.configuration['vimeo'] && @course.school.configuration['vimeo']['access_token'] ||
-          Rails.application.secrets.vimeo_access_token
-        token.present?
+        @vimeo_access_token ||= begin
+          token = @course.school.configuration['vimeo'] && @course.school.configuration['vimeo']['access_token'] ||
+            Rails.application.secrets.vimeo_access_token
+          token.present?
+        end
+      end
+
+      def vimeo_account_plan
+        return '' unless vimeo_access_token?
```

Comment:
Wouldn't a `nil` return be a better option here? So that Rescript can treat this as an `option<string>`?

```suggestion
        return unless vimeo_access_token?
```